standard_test.go

```diff
@@ -765,56 +765,64 @@ func TestEliminateSnakes(t *testing.T) {
 	tests := []struct {
 		Snakes                   []Snake
 		ExpectedEliminatedCauses []string
+		ExpectedEliminatedBy     []string
 		Err                      error
 	}{
 		{
 			[]Snake{},
 			[]string{},
+			[]string{},
 			nil,
 		},
 		{
 			[]Snake{
 				Snake{},
 			},
 			[]string{NotEliminated},
+			[]string{""},
 			errors.New("snake is length zero"),
 		},
 		{
 			[]Snake{
-				Snake{Body: []Point{{1, 1}}},
+				Snake{ID: "1", Body: []Point{{1, 1}}},
 			},
 			[]string{EliminatedByStarvation},
+			[]string{""},
 			nil,
 		},
 		{
 			[]Snake{
-				Snake{Health: 1, Body: []Point{{1, 1}}},
+				Snake{ID: "1", Health: 1, Body: []Point{{1, 1}}},
 			},
 			[]string{NotEliminated},
+			[]string{""},
 			nil,
 		},
 		{
 			[]Snake{
-				Snake{Health: 1, Body: []Point{{-1, 1}}},
+				Snake{ID: "1", Health: 1, Body: []Point{{-1, 1}}},
 			},
 			[]string{EliminatedByOutOfBounds},
+			[]string{""},
```

Comment:
What do you think of setting this to yourself in this case too? I'm 50/50 on it.

Reply:
I have not strong feelings either way on this one.

Reply:
Let's leave it for now, if we want we can add it later.